_pages/en/strategy-build-vs-buy.md

```diff
@@ -110,26 +110,27 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Governance, Compliance, and Reporting
 
 1. IT Solutions are broken down into manageable parts (IT Products)
-2. Each IT Product must be assessed on its own as well as a part of overall the IT Solution.
-3. IT Products may be bought, built, or adopted (e.g. Open Source Software)
-4. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
+2. IT Solutions are assessed against the IT Solutions Risk Assessment Framework (see [Risk Managing the Obtention of Software](#risk-managing-the-obtention-of-software)).
   As such:
    1. Complex and core to mandate IT Solutions are architected and delivered internally by ESDC employees but may leverage third party IT Products as components.
    2. Non-complex and not core to mandate IT Solutions may be architected and delivered by third party integrators and/or vendors.
-5. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
+3. IT Solutions must provide their overall and individual IT products metrics internally to ESDC. Metrics are comprised of the following:
```

Comment:
A question more than a comment. Are these metrics defined anywhere or meant to just be examples. 

What does API consumption statistics mean ? 

What does Integration testing results mean ? End to End testing, unit testing or some other IITB lingo I'm not aware of. 

What does Production deployment audit traces mean ? 

The only thing that makes sense to me here is Accessibility test results, Security and Availability. I am unclear on all the other metrics